app/views/password/_password_recovery_form.erb

```diff
@@ -3,7 +3,7 @@
 <%= form_for @form, url: reset_password_path do |f| %>
   <fieldset class="govuk-fieldset" aria-describedby="changed-password-hint">
     <span id="changed-password-hint" class="govuk-hint">
-      <%= t 'password.reset_password_legend' %>
+      <%= @reset_by_admin ? t('password.admin_reset_password_legend') : t('password.reset_password_legend') %>
```

Comment:
you could also access the `params` directly from here...

Reply:
Do we not want to keep the logic in the controller or is this acceptable to put straight in the view? `@reset_by_admin = params[:reset_by_admin].present?`

Reply:
in this case it would be perfectly fine in the view. There's no logic involved as such, it's just a simple variable check... but really doesn't matter ;)